code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/notification/templates/ProductCatalogPushTemplate.kt

```diff
@@ -0,0 +1,151 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.notification.templates
+
+import android.content.Intent
+import com.adobe.marketing.mobile.services.Log
+import com.adobe.marketing.mobile.services.ui.notification.PushTemplateConstants
+import com.adobe.marketing.mobile.util.DataReader
+import org.json.JSONArray
+import org.json.JSONException
+
+internal class ProductCatalogPushTemplate : AEPPushTemplate {
+    internal var intentAction: String? = null
```

Comment:
Is this same as `ctaButtonUri`? Don't see it assigned in the map constructor

Reply:
this was a copy/paste error... removed it